src/PhoneNumber/PhoneNumber.js

```diff
@@ -84,73 +57,29 @@ class PhoneNumber extends Component {
   };
 
   handleChange = e => {
-    const { value: currentNumber, selectionStart: caretPosition } = e.target;
-    const { phonePattern, value: selectedCountry } = this.getCountryFrom(
-      currentNumber
-    );
+    const { value: currentNumber } = e.target;
+    const { dialingCode } = this.state;
 
-    let formattedNumber = currentNumber.replace(/(?!^\+)\D/gm, '');
+    const phoneNumber = currentNumber.replace(/[^\d]/gm, '');
 
-    if (phonePattern) {
-      formattedNumber = this.formatNumber(formattedNumber, phonePattern);
-    }
+    const formattedNumber = dialingCode
+      ? `+${dialingCode} ${phoneNumber}`
+      : phoneNumber;
 
-    this.setState({ selectedCountry, formattedNumber }, () => {
+    this.setState({ phoneNumber }, () => {
       this.sendChange(formattedNumber);
-      this.setCaretPosition(caretPosition, currentNumber, formattedNumber);
     });
   };
 
-  setCaretPosition = (currentPosition, currentNumber, newPhoneNumber) => {
-    const currentSeparators = this.getSeparatorsToPosition(
-      currentNumber,
-      currentPosition
-    );
-
-    const newSeparators = this.getSeparatorsToPosition(
-      newPhoneNumber,
-      currentPosition
-    );
+  handleCountryClick = dialingCode => {
```

Comment:
I'm not sure about this function's name, maybe it should be handlePrefixClick now that we are not using a CountrySelector?

Reply:
Ok. It makes sense